src/platform_impl/windows/window.rs

```diff
@@ -621,6 +621,41 @@ impl Window {
         warn!("`Window::set_ime_position` is ignored on Windows")
     }
 
+    #[inline]
+    pub fn request_user_attention(&self, request_type: Option<RequestUserAttentionType>) {
+        let window = self.window.clone();
+
+        self.thread_executor.execute_in_thread(move || unsafe {
+            let mut flash_info = match request_type {
+                Some(req_type) => winuser::FLASHWINFO {
+                    cbSize: mem::size_of::<winuser::FLASHWINFO>() as UINT,
+                    hwnd: window.0,
+                    dwFlags: match req_type {
+                        RequestUserAttentionType::Critical => {
+                            winuser::FLASHW_ALL | winuser::FLASHW_TIMERNOFG
+                        }
+                        RequestUserAttentionType::Informational => {
+                            winuser::FLASHW_TRAY | winuser::FLASHW_TIMERNOFG
+                        }
```

Comment:
Could your request to a platform specific variants instead of inlining it here? You should do that
for `uCount` and `dwFlags` for both `None` and `Some` pathes.

Reply:
How do you mean exactly?

Reply:
```rust
    let dwFlags = request_type
        .map(|ty| match ty {
            RequestUserAttentionType::Critical => winuser::FLASHW_ALL | winuser::FLASHW_TIMERNOFG,
            RequestUserAttentionType::Informational => {
                winuser::FLASHW_TRAY | winuser::FLASHW_TIMERNOFG
            }
        })
        .unwarp_or(winuser::FLASHW_STOP);
```